app/views/devise/mailer/password_change.html.erb

```diff
@@ -0,0 +1,3 @@
+<p>Hello <%= @resource.email %>!</p>
```

Comment:
Do we need these other email templates? For example the one about locking would presumably only be used if we had the `lockable` module enable. If there's anything that will never get sent let's get rid of it to avoid confusion.